packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -216,76 +176,32 @@
+    line-height: var(--terra-time-input-time-spacer-line-height, 1.375);
     margin-bottom: var(--terra-time-input-time-spacer-margin-bottom, 0.3571428571rem);
     margin-left: var(--terra-time-input-time-spacer-margin-left, 0.14285714rem);
     margin-right: var(--terra-time-input-time-spacer-margin-right, 0.14285714rem);
     margin-top: var(--terra-time-input-time-spacer-margin-top, 0.3571428571rem);
   }
 
-  .mobile-time-picker {
-    display: flex;
-
-    .time-spacer {
-      line-height: var(--terra-time-input-mobile-line-height, 1.429);
-      margin-top: var(--terra-time-input-time-mobile-spacer-margin-top, 0.3571428571rem);
-    }
-  }
-
   .meridiem-button-group {
-    margin-left: var(--terra-time-input-meridiem-button-group-margin-left, 0.857rem);
-  }
-
-  .mobile-input-label {
-    color: var(--terra-time-input-mobile-label-color, rgba(28, 31, 33, 0.63));
-    font-size: var(--terra-time-input-mobile-label-font-size, 0.857rem);
-  }
-
-  .meridiem-display {
-    background-color: var(--terra-time-input-desktop-background-color, transparent);
-    border: 0;
-    border-radius: var(--terra-time-input-desktop-input-border-radius, 2px);
-    box-sizing: border-box;
-    color: var(--terra-time-input-color, #1c1f21);
-    display: inline-block;
-    height: var(--terra-time-input-desktop-input-height, 1.5714285714rem);
-    margin-left: var(--terra-time-input-meridiem-display-margin-left, 0.1428571429rem);
-    padding-bottom: var(--terra-time-input-meridiem-padding-bottom, 0);
-    padding-left: var(--terra-time-input-meridiem-padding-left, 0);
-    padding-right: var(--terra-time-input-meridiem-padding-right, 0);
-    padding-top: var(--terra-time-input-meridiem-padding-top, 0);
-    text-align: center;
-    width: auto;
-
-    &:hover {
-      border: 0;
-    }
-
-    &.focused {
-      background-color: var(--terra-time-input-desktop-focused-input-background-color, rgba(139, 194, 249, 0.3));
-      border: 0;
-    }
-
-    &[disabled] {
-      background-color: transparent;
-      border: 0;
-    }
+    margin-bottom: var(--terra-time-input-meridiem-button-group-margin-bottom, 0.5rem);
+    margin-right: var(--terra-time-input-meridiem-button-group-margin-right, 0.5rem);
   }
 
   .meridiem-button {
-    background-color: var(--terra-time-input-meridiem-button-background-color, #dedfe0);
+    background-color: var(--terra-time-input-meridiem-button-background-color, transparent);
     border-color: var(--terra-time-input-meridiem-button-border-color, #c8cacb);
     color: var(--terra-time-input-meridiem-button-color, #1c1f21);
     font-size: var(--terra-time-input-meridiem-button-font-size, 1rem);
-    height: var(--terra-time-input-mobile-height, 2.1428571429rem);
-    line-height: var(--terra-time-input-mobile-line-height, 1.429);
+    height: var(--terra-time-input-meridiem-button-height, 2.1428571429rem);
+    line-height: var(--terra-time-input-meridiem-button-line-height, 1.67412);
```

Comment:
It looks like the a.m./p.m. text is not horizontally center aligned and the timezone button is not aligned with the meridiem buttons or time input.

<img width="831" alt="Screen Shot 2019-08-30 at 4 37 11 PM" src="https://user-images.githubusercontent.com/22618655/64052983-288d2a00-cb46-11e9-8906-f593feb98acb.png">

Reply:
You're right, Ben. I am waiting on UX to validate these new values and I shall follow-up.

@neilpfeiffer 👋

Reply:
The a.m./p.m.text is aligned with the baseline of the time input:
![Screen Shot 2019-09-20 02 56 55 PM](https://user-images.githubusercontent.com/726351/65356030-0ae93880-dbb9-11e9-997e-4b5295aaf461.png)

To get the desired alignment, I set the `span` inside each button label to `line-height: 1;`
![Screen Shot 2019-09-20 03 00 35 PM](https://user-images.githubusercontent.com/726351/65356286-ae3a4d80-dbb9-11e9-8290-dc4b63df51ad.png)

I'm also seeing that there are some really funky alignment issues in Firefox:
![Screen Shot 2019-09-17 10 51 07 AM](https://user-images.githubusercontent.com/726351/65356313-bf835a00-dbb9-11e9-8596-276087d870ad.png)

Reply:
@nramamurth @jeremyfuksa  Are we still waiting for UX feedback? Looks like the TZ and meridiem buttons are still misaligned.

Reply:
@benbcai @nramamurth setting `line-height: 1;` within each button label's `span` was the UX feedback. That will make everything align against centers rather than the text baseline.

Reply:
@benbcai I’m sorry, I seem to have missed your comment over the weekend. I shall work on this and update the upgrade guides as well so it’s ready for MVB.

Reply:
@benbcai @jeremyfuksa 

Here's a couple of findings: 
1. While setting `line-height: 1;` for the meridiem buttons solves the horizontal alignment issue in them, it shifts the button group down making it misalign with the time-input group. 

<details> <summary> <b> With `line-height: 1;` for the meridiem button </b> </summary> 

<img width="1163" alt="Screen Shot 2019-10-01 at 4 10 58 PM" src="https://user-images.githubusercontent.com/10090859/65955679-4eb82980-e466-11e9-97e7-eac12abcc4f7.png">
</details>

--- 
2. To fix the alignment w.r.t. the TZ (offset) button, I introduced a line-height of 2.16285 to the [offset button](https://github.com/cerner/terra-framework/blob/7734543bf6304d7a4a4c6319fbdcc1b4f2ea4684/packages/terra-date-time-picker/src/_TimeClarification.module.scss#L55-L62). While it seemed to have fixed the issue on Safari and Chrome, the _wdio_ screenshot shows misalignment - notice that the TZ button has moved up a bit. I suspect the tests run against headless browser and I can't seem to find out the root cause. Do they? 

<details> <summary> <b> Chrome (current line-height) ❌ </summary> 

![Screen Shot 2019-10-01 at 4 07 28 PM](https://user-images.githubusercontent.com/10090859/65955815-92129800-e466-11e9-8563-a9a1830c75fa.png)
</details>

<details> <summary> <b> Chrome (updated line-height 2.16285) ✅ </b></summary> 

![Screen Shot 2019-10-01 at 4 06 52 PM](https://user-images.githubusercontent.com/10090859/65955862-b7070b00-e466-11e9-801c-d8b5e09286a3.png)
</details>

<details> <summary> <b> Headless (?) (current line-height) ✅ </b></summary> 

![Displays_Offset_button_after_clicking_daylight_button 1  2](https://user-images.githubusercontent.com/10090859/65956028-0e0ce000-e467-11e9-840b-1cab1dba0b74.png)
</details>

<details> <summary> <b> Headless (?) (updated line-height 2.16285) ❌ </b></summary> 

![Displays_Offset_button_after_clicking_daylight_button 1](https://user-images.githubusercontent.com/10090859/65955987-f7668900-e466-11e9-947e-6d301e368dd5.png)
</details>

---

And Firefox is behaving totally funky.

Reply:
When you say "current line-height", I'm assuming you mean `line-height: 1`, correct?

Reply:
@jeremyfuksa If you are talking about the TZ button, then no, its current line-height is the default line-height that is 1.429 that comes from terra-button.

Reply:
Using relative positioning on the meridiem button group and then offsetting the top by -1px fixes the styling, but that's not 100% ideal.

Reply:
@jeremyfuksa It is already relative. I have however seemed to have fixed the alignment w.r.t. the TZ button. It'd be great if you could validate these changes in the latest deployment and this has resolved the funkiness in Firefox as well. 

Also, I think that the "a.m./p.m." text "misalignment" is probably because they are lowercase text. I changed it to uppercase just for internal testing and they seem to be aligned correctly to me. So, I am assuming that it is okay? 

<img width="277" alt="Screen Shot 2019-10-07 at 11 12 20 AM" src="https://user-images.githubusercontent.com/10090859/66288421-c539a900-e8f6-11e9-8b6a-b23e7a00a412.png">

cc @benbcai